id-repository/credential-service/src/main/java/io/mosip/credentialstore/provider/CredentialProvider.java

```diff
@@ -247,6 +250,22 @@ public Map<AllowedKycDto, Object> prepareSharableAttributes(IdResponseDTO idResp
 						attributesMap.put(key, formattedObject);
 					} else if (attribute.equalsIgnoreCase(CredentialConstants.ENCRYPTIONKEY)) {
 						additionalData.put(key.getAttributeName(), credentialServiceRequestDto.getEncryptionKey());
+					}else if(attribute.equalsIgnoreCase(CredentialConstants.VID)){
+						VidInfoDTO vidInfoDTO=null;

```

Comment:
You don't need to assign to null, you can just declare it like `VidInfoDTO vidInfoDTO;`

Reply:
taken care